README.md

```diff
@@ -1,11 +1,9 @@
-[![Maven Central](https://img.shields.io/maven-central/v/io.camunda/zeebe-process-test)](https://search.maven.org/search?q=g:io.camunda%20a:zeebe-process-test)
+[![Maven Central](https://img.shields.io/maven-central/v/io.camunda/zeebe-process-test-root)](https://search.maven.org/search?q=g:io.camunda%20zeebe-process-test)
```

Comment:
```suggestion
[![Maven Central](https://img.shields.io/maven-central/v/io.camunda/zeebe-process-test-root)](https://search.maven.org/search?q=g:io.camunda%20zeebe-process-test-root)
```

Reply:
I did this on purpose. The difference is the list of dependencies you will see when you click on the badge. If you put `zeebe-process-test-root` here, you will only see this dependency. By setting it to `zeebe-process-test` it will give a list of all `zeebe-process-test*` dependencies that are in the `io.camunda` group.